query-connector/src/app/queryBuilding/dataState/UserQueriesDisplay.tsx

```diff
@@ -15,25 +26,35 @@ interface UserQueriesDisplayProps {
  * @returns the UserQueriesDisplay to render the queries with edit/delete options
  */
 export const UserQueriesDisplay: React.FC<UserQueriesDisplayProps> = ({
-  queries,
+  queries: initialQueries,
 }) => {
   const router = useRouter();
+  const context = useContext(DataContext);
+  const [queries, setQueries] = useState<CustomUserQuery[]>(initialQueries);
   const [loading, setLoading] = useState(false);
-
-  const handleClick = async () => {
-    setLoading(true);
-
-    // Redirect to query updating/editing page
-    router.push("/queryBuilding/buildFromTemplates");
-  };
+  const modalRef = useRef<ModalRef>(null);
+  const [selectedQuery, setSelectedQuery] = useState<{
+    queryName: string;
+    queryId: string;
+  } | null>(null);
 
   return (
     <div>
+      <ToastContainer position="bottom-left" />
```

Comment:
We should probably put this into the `page.tsx` component should any of the subsequent pages in this route need toasts. Think the reason why we couldn't get the existing code housed in design system is because this higher-order wrapper wasn't there.

For a follow up PR: we should probably have a single page component across all our existing / future routes that pages can inherit from (that would include things like this ToastContainer, stylings for default page widths, etc.)

Reply:
moved the toast to `page.tsx` and created follow-up ticket: https://linear.app/skylight-cdc/issue/QUE-109/update-routing-for-formattingreused-components